codecov.yml

```
@@ -23,24 +23,20 @@ ignore:
 
 coverage:
   status:
-    patch:
-      default:
-        target: auto  # target coverage is equal to the PR base
-        threshold: 0% # coverage is not allowed to reduce vs. the PR base
-        base: auto
-        informational: true
-        enabled: true
     project:
       default:
         informational: true
+    patch:
+      contracts:
+        base: auto
+        target: auto
+        threshold: 0%
```

Think you meant 100 here? Though maybe 95 is better, some branches can be really hard to get coverage for when they are never intended to be hit (e.g. invariants). Or, we can start at 100, and loosen if it becomes a blocker
```suggestion
        threshold: 100%
```

---

I definitely lean in favor of not 100% because universally not all loc will need to be tested and then it creates this artificial headache blocking merge

---

Let's start at 100% and loosen if it's a blocker. Codecov is backwards and this threshold represents how much coverage is allowed to drop, so 0% is correct here.

---

Can we agree on loosening this to 95%?

---

Approved the PR, pending this change

---

You own this code so you can do whatever you want here, I do think 100% is not a good idea and would prefer 95%

---

https://about.codecov.io/blog/the-case-against-100-code-coverage/

>Although there is no perfect code coverage value, we can still make a stab at such a value. 2 years ago, Google posted a [blog post](https://testing.googleblog.com/2020/08/code-coverage-best-practices.html) on their best practices:
“We offer the general guidelines of 60% as “acceptable”, 75% as “commendable” and 90% as “exemplary.””
At Codecov, we see that a significant portion of our open source repositories centers around 80% coverage.

---

Ok let's start with 95% and see what happens. We can adjust if necessary, will be flexible with this.

---

@tynes can you approve in that case?